SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -27,6 +28,8 @@ enum class ESpatialConnectionType
 	Locator
 };
 
+typedef std::function<bool(const Worker_Alpha_LoginTokensResponse*)> LoginTokenCb;
```

Comment:
Style guide - `using` instead of `typedef` and `TFunction` instead of `std::function`.